packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -74,7 +75,7 @@ export function SquadPageHeader({
         onNewSquadPost={onNewSquadPost}
         className={sidebarRendered && 'absolute top-0 right-[4.5rem]'}
       />
-      {hasTriedOnboarding && (
+      {enableNotifications.isActive && (
```

Comment:
This would mean the user would only see the notification banner after closing the popup. Which is correct on the first time, but what about on the second time? Should we now keep it persistent to be displayed since they already saw the SquadTour?